File: OmkarFCUBEAPI/FleetTrans/Models/DieselStatementModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class DieselStatementModel
    {
        public string? MasterID { get; set; }
        public string? DfVendor { get; set; }
        public string? BillStmtNo { get; set; }
        public string? BillStmtDate { get; set; }
        public string? FromDate { get; set; }
        public string? ToDate { get; set; }
        public string? Location { get; set; }
        public string? Remarks { get; set; }
        public string? TotalDslLtrs { get; set; }
        public string? TotalCashAdv { get; set; }
        public string? TotalDslAmt { get; set; }
        public string? TotalNetAmount { get; set; }
        public string? BranchCode { get; set; }
        public string? YearId { get; set; }
    }
}

File: OmkarFCUBEAPP/src/app/models/dieselstatementmodel.ts
export class Dieselstatementmodel {
    masterID: string = "";
    dfVendor: string = "";
    vendor: string = "";
    billStmtNo: string = "";
    billStmtDate: string = "";
    fromDate: string = "";
    toDate: string = "";
    location: string = "";
    transType:string = "";
    rate: string = "";
    statementFlag: string = "";
    remarks: string = "";
    totalDslLtrs: string = "";
    grossDslAmt: string = "";
    discRateLtr: string = "";
    discAmt: string = "";
    totalDslAmt: string = "";
    tdsRate: string = "";
    tdsAmt: string = "";
    totalCashAdv: string = "";
    totalNetAmount: string = ""; 
    branchCode: string = "";
    yearId: string = "";
    loggedInUser: string = "";
    
    dieselStatementListData: Dieselstatementsearchmodel[]=[];
}

export class Dieselstatementsearchmodel {
    pmtId: string = "";
    branch: string = "";
    pmtDate: string = "";
    vehicleNo: string = "";
    hsdAdvType: string = "";
    transDesc: string = "";
    qtyLtrs: string = "";
    ratePerLtr: string = "";
    amountPaid: string = "";
    remarks: string = "";
    selected: boolean = false;
}




